### PR TITLE
Add description and authors

### PR DESCRIPTION
**Pull request by georgegevoian**

### CODE REVIEW

File: dropbox-embed/package.json [hunk 1]
```diff
@@ -9,6 +9,12 @@
     "widgetId": "@gristlabs/dropbox-embed",
     "published": true,
     "accessLevel": "read table",
-    "renderAfterReady": true
+    "renderAfterReady": true,
+    "description": "View and access files from your Dropbox account.",
+    "authors": [
+      {
+        "name": "Grist Labs"
```
Comment (berhalak) [inline dropbox-embed/package.json:16]: Not sure it was our.

File: pivottable/package.json [hunk 2]
```diff
@@ -9,6 +9,12 @@
     "widgetId": "@gristlabs/widget-pivottable",
     "published": true,
     "accessLevel": "read table",
-    "renderAfterReady": true
+    "renderAfterReady": true,
+    "description": "Build and modify a pivot table from a selected Grist table.",
+    "authors": [
+      {
+        "name": "Grist Labs"
```
Comment (berhalak) [inline pivottable/package.json:16]: This is not ours.

File: qrcode/package.json [hunk 3]
```diff
@@ -10,7 +10,13 @@
       "widgetId": "@gristlabs/widget-qrcode",
       "published": true,
       "accessLevel": "read table",
-      "renderAfterReady": true
+      "renderAfterReady": true,
+      "description": "Generate custom QR codes based on data from a selected record.",
+      "authors": [
+        {
+          "name": "Grist Labs"
```
Comment (berhalak) [inline qrcode/package.json:17]: Not ours.

**Review by berhalak — COMMENTED** [review]
Looks ok, but couple of widgets were created by the community. Not sure how to change them though.

File: dropbox-embed/package.json (hunk 1 above)
Comment (georgegevoian) [reply dropbox-embed/package.json:16]: Looks like Dmitry contributed this one.

**Comment by georgegevoian** [comment]
> Looks ok, but couple of widgets were created by the community. Not sure how to change them though.
Should be fixed now. Thanks.

**Review by berhalak — APPROVED** [review]

**Pull request merged**